consumer/dragonboard/dragonboard410c/installation/board-recovery.md

```diff
@@ -35,9 +35,9 @@ In order to force the DB410c to boot on USB (EDL mode), you need to configure S6
 
 * Power off the board and make sure no USB cable is plugged into the board
 * Set switch S6-1 to `ON`.
-* Connect the debug UART / serial console to your Linux PC, if not done already
 * Connect the micro USB cable (J4) between the Linux PC and the board
-* Open UART/serial console
+* Connect the debug UART / serial console to your Linux PC (optional)
+* Open UART/serial console (optional)
```

Comment:
Why keep these at all? No part of the recovery instructions gives an information about what to expect on the serial port if you (optionally) connect it. It's probably better just not to mention it.

Reply:
I thought about it.... But since we have used UART in other boards as well (db820c, Hikey960), I decided to keep it. Anyway, if @ndechesne is not having any issue with it, I can remove.

Reply:
IHMO both the db820c and hikey960 documents are faulty! Both require the user to connect a UART adapter that is not used or needed and which distract attention from the stdout of the flashing tool.